src/clients/rust/Cargo.lock

```diff
@@ -328,9 +203,9 @@ dependencies = [
 
 [[package]]
 name = "windows-sys"
-version = "0.52.0"
+version = "0.59.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "282be5f36a8ce781fad8c8ae18fa3f9beff57ec1b52cb3de0789201425d9a33d"
+checksum = "1e38bc4d79ed67fd075bcc251a1c39b32a1776bbe92e5bef1f0bf1f8c531853b"
 dependencies = [
  "windows-targets",
 ]
```

Comment:
🤔 do we need to commit a lockfile? I am personally a fan of the original guideline that libraries shouldn't commit lockfiles. It makes CI a bit more wobbly, but that matches the experience of the downstream consumer, as _they_ won't use our lockfile, and puts pressure on us to avoid wobbly dependencies!